code.gs

```diff
@@ -156,6 +168,13 @@ var settings = {
      * Choose a date you know should trigger an event notification.
      */
     testDate: new Date('2017/10/19 06:00:00')
+  },
+  developer: {
+    /* NB: Users shouldn't need to (or want to) touch these settings. They are here for the
+     *     convenience of developers/maintainers only.
+     */
+    version: '3.2.0-alpha',
```

Comment:
Moving the version number from [line L845](https://github.com/GioBonvi/GoogleContactsEventsNotifier/pull/60/files#diff-deb2c49f6ef99255c9830fe80e966b46L845) to here requires [line R1173](https://github.com/GioBonvi/GoogleContactsEventsNotifier/pull/60/files#diff-deb2c49f6ef99255c9830fe80e966b46R1173) to be modified with an updated regex.

However, since this would not be the first time that moving the version number around breaks this functionality for old versions, I think we should be using [GitHub's API](https://developer.github.com/v3/repos/releases/#get-the-latest-release) instead. (In fact I don't know why I did not think of this to start with...).  
The latest version number would be obtained simply by GETting and parsing this JSON file: https://api.github.com/repos/GioBonvi/GoogleContactsEventsNotifier/releases/latest which should be both more reliable and stable across releases and would still let use move the current version number around as we like.

Let me know what you think of this: if you want and have the time you can implement this in a new commit in this PR, otherwise I'll do it once this PR is merged.